### PR TITLE
fix: 2 datasets for Georgian language

### PR DESCRIPTION
**Pull request by jupyterjazz**
## Checklist for adding MMTEB dataset

Two datasets for Georgian language:
1. `GeorgianFAQRetrieval` - Frequently asked questions (FAQs) and answers mined from Georgian websites via Common Crawl.
2. `TbilisiCityHallBitextMining` - Parallel news titles from the Tbilisi City Hall website

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/BitextMining/kat/TbilisiCityHallBitextMining.py
Comment (shreeya-dhakal) [inline mteb/tasks/BitextMining/kat/TbilisiCityHallBitextMining.py:46]: Please set:
license="Not specified"
and add avg_character_length.

**Review by isaac-chung — COMMENTED** [review]
Looks good. Please add the missing fields @shreeya-dhakal mentioned and add points for the review.

**Comment by jupyterjazz** [comment]
Hey @shreeya-dhakal and @isaac-chung, few questions about points:
`GeorgianFAQRetrieval` seems to be the first Georgian Retrieval dataset, so should I add bonus 4 points? 
Also, I added couple of Spanish datasets in this group PR https://github.com/embeddings-benchmark/mteb/pull/227, will this count?

**Comment by jupyterjazz** [comment]
@isaac-chung I think it exists but all points were automatically assigned to the main author

**Comment by isaac-chung** [comment]
For multiple-author PRs, we defer to the authors on how to divide the points amongst them. Please coordinate with your co-author(s) on how you want to split the points. We can use a different PR to address them, as this will be good to merge after you add pts for the bonus and review.

**Review by isaac-chung — APPROVED** [review]

**Pull request merged**
